extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -425,10 +425,19 @@ export class Bind {
    */
   initialize_(root) {
     dev().info(TAG, 'init');
-    return Promise.all([
-      this.addMacros_(),
-      this.addBindingsForNodes_([root]),
-    ]).then(results => {
+
+    // Disallow URL property bindings in AMP4EMAIL.
+    const allowUrlProperties = !this.isAmp4Email_();
```

Comment:
I really think the allowing of url properties should be declarative and defined in the markup rather than forcing the spec to comply with this rule. This would be more flexible. I think this capability should be configured similarly with the way allow-xhr-interception is set on the html tag.

Reply:
This behavior is already defined in the validator since `[href]` and similar attributes are not whitelisted in the AMP4EMAIL spec. This PR just makes amp-bind's runtime validator (which is basically a forked subset) consistent with the AMP validator.